pixart.js

```diff
@@ -1 +1,21 @@
+var button = document.querySelector("button");
+var brushBox = document.querySelector(".brush");
+var colorBox = document.querySelector("#color-field");
 
+button.addEventListener( "click", function() {
+  event.preventDefault();
+  colorBox.setAttribute("value", "");
+  brushBox.style.background = colorBox.value;
+  paintColor = colorBox.value;
+  return paintColor;
```

Comment:
Returning from an event listener callback function isn't useful, as nothing is paying attention to the return value.